openbts/components.py

```diff
@@ -41,6 +42,37 @@ def monitor(self):
     }
     return self._send_and_receive(message)
 
+  def tmsis(self, fields=['IMSI', 'ACCESSED'], access_period=0, auth=2):
+    """Gets all active subscribers from the TMSI table.
+
+    Args: access_period: fetches all entries with ACCESS < acess_period
+                         (default=0 filter off)
+          auth: fetches all entries with AUTH = auth
```

Comment:
They are defined by the [Authorization enum](https://github.com/endaga/openbts/blob/master/Control/TMSITable.h#L45) in the TMSI table. This field tells us whether a MS is authorized to use the network.The reason for allowing filtering on AUTH is that the TMSI table can actually get quite full with non authorized users, so this field allows developers to fetch the subscribers they care about. I should probably update the docstring to list out the different values since they are not listed in OpenBTS docs.